File: FrontEnd/src/app/features/sales/documentTypes/classes/view-models/documentType-list-vm.ts
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface DocumentTypeListVM {

    id: number
    shipOwner: SimpleEntity
    ship: SimpleEntity
    description: string
    batch: string
    isActive: boolean
    table8_1: string
    table8_8: string
    table8_9: string

}

File: API/Features/Sales/DocumentTypes/ViewModels/DocumentTypeListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Sales.DocumentTypes {

    public class DocumentTypeListVM {

        public int Id { get; set; }
        public SimpleEntity Ship { get; set; }
        public SimpleEntity ShipOwner { get; set; }
        public string Abbreviation { get; set; }
        public string Description { get; set; }
        public string Batch { get; set; }
        public bool IsActive { get; set; }
        public string Customers { get; set; }
        public string Suppliers { get; set; }
        public bool IsMyData { get; set; }
        public string Table8_1 { get; set; }
        public string Table8_8 { get; set; }
        public string Table8_9 { get; set; }

    }

}